_posts/2018-04-10-building-the-peer-to-peer-internet.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Building The Peer-to-Peer Internet
+slug: building-the-peer-to-peer-internet
+text: Workshop series at the Toronto Public Library
+location: Fort York branch of the Toronto Public Library
+locationLink: https://www.torontopubliclibrary.ca/detail.jsp?Nr=p_cat_branch_name:Fort%20York
+date: 2018-04-10
+startTime: '18:30'
+endTime: '20:00'
+---
+
+Toronto Mesh will be presenting the workshop series **{{ page.title }}** each Tuesday from April 10 to May 15, 2018 at the [{{ page.location }}]({{ page.locationLink }}).
```

Comment:
s/presenting/facilitating (hosting? running?)

Reply:
Will change to `running`